debezium-connect-rest-extension/src/test/java/io/debezium/kcrestextension/TestHelper.java

```diff
@@ -47,18 +47,16 @@ public static DebeziumContainer getDebeziumContainer() {
     }
 
     public static void setupDebeziumContainer(String debeziumContainerImageVersion) {
-        DEBEZIUM_CONTAINER = new DebeziumContainer(DockerImageName.parse("quay.io/debezium/connect:" + debeziumContainerImageVersion))
+
+        DEBEZIUM_CONTAINER = new DebeziumContainer(DockerImageName.parse("debezium/connect-base:" + debeziumContainerImageVersion))
```

Comment:
Because for the tests where Kafka Connect does not support topic auto-creation (prior 2.6.0) we are using ancient Debezium 1.2 which is not available on `quay.io`. This code here even changed drastically in my workspace (not pushed yet) to enable some more tests and needs to be reviewed later again after I finished adding those additional tests.